app/views/shared/_translations.html.erb

```diff
@@ -1,7 +1,6 @@
-<% if @content_item.available_translations.any? %>
+<% if @content_item.available_translations.any? && @content_item.available_translations.length > 1 %>
```

Comment:
Why do we need both `@content_item.available_translations.any?` and `@content_item.available_translations.length > 1`?